core/src/validator.rs

```diff
@@ -150,8 +150,8 @@ const WAIT_FOR_WEN_RESTART_SUPERMAJORITY_THRESHOLD_PERCENT: u64 =
 #[derive(Clone, EnumString, EnumVariantNames, Default, IntoStaticStr, Display)]
 #[strum(serialize_all = "kebab-case")]
 pub enum BlockVerificationMethod {
-    #[default]
     BlockstoreProcessor,
+    #[default]
```

Comment:
I just wanted to see there's strange breakage in local-clusters. seems not. :)

i'm still thinking whether to enable unified scheduler by default straightly.

Reply:
> i'm still thinking whether to enable unified scheduler by default straightly.

i decided. and pr description got updated.